calcom/cal.com
# Use the same locale for event description.

Currently all headers  of the event description use organizer's language except description part. This is not suitable to see different languages withing the same text block.



## What does this PR do?

Change description header to use the same locale as others.

## Type of change




## How should this be tested?

Create an event for attendees in a different locale. When they receive an event in google calendar check the description. Currently it has the following structure (if on of the 1 has ru locale):
What:
Who:
Where:
Описание:

The fix would change it to:
What:
Who:
Where:
Description:
